src/main/resources/META-INF/plugin.xml

```diff
@@ -21,10 +21,15 @@
         <applicationService
                 serviceImplementation="nl.tudelft.ewi.se.ciselab.testgenie.toolwindow.TestGenieToolWindowService"/>
         <applicationConfigurable
-                parentId="Settings"
-                instance="nl.tudelft.ewi.se.ciselab.testgenie.settings.TestGenieSettingsConfigurable"
-                id="nl.tudelft.ewi.se.ciselab.testgenie.settings.TestGenieSettingsConfigurable"
-                displayName="TestGenie Advanced Settings"/>
+                parentId="tools"
+                instance="nl.tudelft.ewi.se.ciselab.testgenie.settings.SettingsPluginConfigurable"
+                id="TestGenieSettings"
+                displayName="TestGenie"/>
```

Comment:
This change will cause problems with the link in the quick access tool window tab. 

First of all note that there are merge conflicts in your pull request. The conflicts are most likely caused because of this change in plugin.xml file. If you resolve the changes, make sure that you also test out the link in the quick access tool window tab. 

Also you will probably need to decide to which settings (EvoSuite or TestGenie) the link should take you to.

Reply:
I think that should work. Good.